### PR TITLE
Adding template and openapi schema for upload file handlers

### PR DESCRIPTION
**Pull request by truhacevkir**
Adding openapi spec and template configuration for multipart-upload handler.
- Not adding publicationIdentifier in path, should be added once it is used.

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -1090,6 +1109,101 @@ Resources:
...
+            RestApiId: !Ref NvaPublicationApi
+
+  AbortUploadFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: publication-file
+      Environment:
+        Variables:
+          NVA_PERSISTED_STORAGE_BUCKET_NAME: !Ref ResourceStorageBucketName
+          ALLOWED_ORIGIN: '*'
```
Comment (torbjokv) [inline template.yaml:1177]: !Ref AllowedOrigins instead maybe?

File: template.yaml [hunk 2]
```diff
@@ -1090,6 +1109,101 @@ Resources:
...
+            RestApiId: !Ref NvaPublicationApi
+
+  CompleteUploadFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: publication-file
+      Environment:
+        Variables:
+          NVA_PERSISTED_STORAGE_BUCKET_NAME: !Ref ResourceStorageBucketName
+          ALLOWED_ORIGIN: '*'
```
Comment (torbjokv) [inline template.yaml:1196]: !Ref AllowedOrigins instead maybe?

File: template.yaml [hunk 3]
```diff
@@ -1090,6 +1109,101 @@ Resources:
...
+            RestApiId: !Ref NvaPublicationApi
+
+  ListPartsUploadFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: publication-file
+      Environment:
+        Variables:
+          NVA_PERSISTED_STORAGE_BUCKET_NAME: !Ref ResourceStorageBucketName
+          ALLOWED_ORIGIN: '*'
```
Comment (torbjokv) [inline template.yaml:1139]: !Ref AllowedOrigins instead maybe?

File: template.yaml [hunk 4]
```diff
@@ -1090,6 +1109,101 @@ Resources:
...
             RestApiId: !Ref NvaPublicationApi
+            
+  CreateUploadFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: publication-file
+      Environment:
+        Variables:
+          NVA_PERSISTED_STORAGE_BUCKET_NAME: !Ref ResourceStorageBucketName
+          ALLOWED_ORIGIN: '*'
```
Comment (torbjokv) [inline template.yaml:1120]: !Ref AllowedOrigins instead maybe?

**Review by brinxmat — APPROVED** [review]

**Pull request merged**